android-build.gradle

```diff
@@ -4,5 +4,5 @@ repositories {
 }
 
 dependencies {
-    api 'com.tencent.mm.opensdk:wechat-sdk-android-without-mta:+'
+    api 'com.tencent.mm.opensdk:wechat-sdk-android-without-mta:6.8.0'
```

Comment:
I wanted to peg this to 6.8.0 instead of 'latest' so it doesn't get upgraded out from under us and break again :)